helm/kubernetes-cluster-prep/templates/tests/_helm_test.bats.txt

```diff
@@ -74,4 +75,45 @@ text_color "$MAGENTA"
   assert_failure
 }
 {{- end }}
+
+@test "Conjur Account is valid" {
+  display_info "Validating account from the info endpoint for enterprise"
+
+  cmd=( curl -k --connect-timeout 5 "$conjurApplianceUrl"/info -o "$TEMP_INFO_FILE")
+  display_info "Running ${cmd[@]}"
+  run "${cmd[@]}"
+  run grep account.*"$conjurAccount" "$TEMP_INFO_FILE"
+
+  if [ "$status" -ne 0 ]; then
+    if (grep account "$TEMP_INFO_FILE"); then
+        status=0
```

Comment:
I think we can eliminate this `status=0` and the `assert_failure` on line 92, and just use an `assert_success` on line 96? Something like this:
```
  if [ "$status" -ne 0 ]; then
    if (grep account "$TEMP_INFO_FILE"); then
        display_error "Please check the configured Conjur Account.\n" \
                      "It does not match with values from the info endpoint"
    else
        skip "test due to the info endpoint is not available for OSS"
    fi
  fi
  assert_success
}
```

Reply:
It seems to need the `status=0` and `assert_failure`, otherwise it doesn't fail.
For the first command the status is set, but on the second command it's not getting
set automatically. I'm not sure why I didn't see it mentioned in the docs.

Reply:
updating per our offline discussion.